### ISSUE TITLE
Fix TypeScript errors

### ISSUE THREAD

**Issue by AlexanderLukin** [amcharts/amcharts3-angular2#15]
Now when I try to compile my project with `amcharts3-angular2` directive, I run into batch of TypeScript compilation issues such as
```
dist\tmp\app\landing\amcharts.directive.ts(15,18): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(21,21): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(26,5): error TS7017: Index signature of object type implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(32,20): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(46,15): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(64,23): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(64,26): error TS7006: Parameter 'y' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(82,24): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(82,27): error TS7006: Parameter 'y' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(105,16): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(109,18): error TS7006: Parameter 'x' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(109,21): error TS7006: Parameter 'y' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(137,31): error TS7006: Parameter 'chart' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(137,38): error TS7006: Parameter 'listeners' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(150,28): error TS7006: Parameter 'chart' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(150,35): error TS7006: Parameter 'oldObj' implicitly has an 'any' type.
dist\tmp\app\landing\amcharts.directive.ts(150,43): error TS7006: Parameter 'newObj' implicitly has an 'any' type.
```

I assume thet those issues should be fixed.

**Comment by Pauan** [comment]
This should be fixed with `amcharts3-angular2` version `1.2.0`, thanks for the report!

**Comment by Pauan** [comment]
You can upgrade to the latest version of `amcharts3-angular2` by deleting your `node_modules/amcharts3-angular2` folder and then running `npm install`
